fantiadl/models.py

```diff
@@ -44,7 +44,7 @@
 POSTS_URL = "https://fantia.jp/posts"
 POST_RELATIVE_URL = "/posts/"
 
-TIMELINES_API = "https://fantia.jp/api/v1/me/timelines/posts?page={}&per=24"
+TIMELINES_API = "https://fantia.jp/api/v1/me/timelines/posts?page={}&per=24{}"
```

Comment:
Remove this, and if necessary refactor the other use of this constant to pass `params`, e.g.:

```python
TIMELINES_API = "https://fantia.jp/api/v1/me/timelines/posts"
PAGE_PER_N = 24

response = self.session.get(TIMELINES_API, params={"page": page_number, "per": PAGE_PER_N})
```

Reply:
LGTM, initially I just tried to minimalize changes to code, passing params is definitely better.
I'm a bit busy recently, these requested  changes might come in 3-7 days.

Reply:
Requested changes have been applied. It's very unlikely that the number 24 will change or anyone would like to change it, so I didn't introduce a new constant.